modules/persistent-storage-csi-cinder-storage-class.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-cinder.adoc
+
+[id="persistent-storage-csi-cinder_{context}"]
+= Making OpenStack Cinder CSI the default storage class
+
+In {product-title}, the default storage class references the in-tree Cinder driver. The storage class will default to referencing OpenStack Cinder CSI in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree storage class are planned for migration to the OpenStack Cinder CSI storage class at that time.
+
+The OpenStack Cinder CSI driver uses the `cinder.csi.openstack.org` parameter key to support dynamic provisioning.
+
+.Procedure
+To enable the use of OpenStack Cinder CSI in {product-title} {product-version}, you must request the `standard-csi` storage class to overwrite the default in-tree storage class.
+
+. Create the `PersistentVolumeClaim` object using the Cinder CSI storage class, as shown in the following example:
++
+[source,yaml]
+--
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cinder-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: standard-csi <1>
+--
+<1> Assigns the default storage class.
+
+. Verify that the PVC is now referencing the CSI storage class by default:
++
+[source,terminal]
+--
+$ oc get storageclass
+--
++
+[source,terminal]
+.Example output
+--
+NAME                   PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+csi-manila-ceph        manila.csi.openstack.org   Delete          Immediate              false                  46h
+standard               kubernetes.io/cinder       Delete          WaitForFirstConsumer   true                   46h
+standard-csi(default)  cinder.csi.openstack.org   Delete          WaitForFirstConsumer   false                  46h
```

Comment:
Just a note that the `standard-csi` storage class should have `ALLOWVOLUMEEXPANSION` to `true` after https://github.com/openshift/openstack-cinder-csi-driver-operator/pull/15. We want to merge this patch for 4.7.

Reply:
Ah, good catch. This is also captured in OCP 4.7 docs in the table here: https://docs.openshift.com/container-platform/4.7/storage/container_storage_interface/persistent-storage-csi.html#csi-drivers-supported_persistent-storage-csi ("CSI resize")